assets/javascripts/job_templates.js

```diff
@@ -157,7 +158,12 @@ function buildMediumGroup(group, media) {
     $.each(testnames, function(ti, test) {
 	var tr = $('<tr class="test_' + test + '"/>').appendTo(tbody);
 	tr.data('test-id', tests[test]['id']);
-	$('<td class="name">' + test + '</td>').appendTo(tr);
+	if (test.length < 70) {
+		shortname = test;
```

Comment:
I guess you don't need `shortname` to be global variable. Can you declare it before `if` block?